src/geo/gmaps/projector.js

```diff
@@ -1,21 +1,30 @@
 /* global google */
-// helper to get pixel position from latlon
-var Projector = function (map) { this.setMap(map); };
-Projector.prototype = new google.maps.OverlayView();
-Projector.prototype.draw = function () {};
-Projector.prototype.latLngToPixel = function (point) {
-  var p = this.getProjection();
-  if (p) {
-    return p.fromLatLngToContainerPixel(point);
+
+function Projector (map) {
+  this._projection = map.getProjection();
+  if (!this._projection) {
+    google.maps.event.addListenerOnce(map, 'projection_changed', function () {
+      this._projection = map.getProjection();
+    });
+  }
+}
+
+Projector.prototype.latLngToPixel = function (latlng) {
```

Comment:
Transform coordinates into pixels without knowing the projection has no sense so I wont return anything is there is no projection.

```javascript
Projector.prototype.latLngToPixel = function (latlng) {
  return this._projection.fromLatLngToPoint(latlng);
};
```

This will throw a better error instead `cannot read property fromLatLngToPoint from undefined`
```javascript
Projector.prototype.latLngToPixel = function (latlng) {
  if(!this._projection) {
    throw new Error('Projector.latLngToPixel needs a projection to work'); 
  }
  return this._projection.fromLatLngToPoint(latlng);
};
```

Reply:
@IagoLast the issue here is that the projection is not defined when the tests are executed. I have checked that tests are failing with your approach, so if we don't return the same point if there is no projection we must update the failing tests